docs/scripting/functions/db_close.md

```diff
@@ -1,58 +1,73 @@
 ---
 id: db_close
 title: db_close
-description: Closes an SQLite database that was opened with db_open.
-tags: [sqlite]
+description: Closes a SQLite database connection that was opened with `db_open`.
+keywords:
+    - sqlite
 ---
 
 :::warning
 
-This function starts with lowercase letter.
+This function starts with a lowercase letter.
 
 :::
 
 ## Description
 
-Closes an SQLite database that was opened with db_open.
+Closes a SQLite database connection that was opened with [db_open](db_open).
 
-| Name  | Description                                                           |
-| ----- | --------------------------------------------------------------------- |
-| DB:db | The handle of the database connection to close (returned by db_open). |
+| Name  | Description                                                                      |
+| ----- | -------------------------------------------------------------------------------- |
+| DB:db | The handle of the database connection to close (returned by [db_open](db_open)). |
 
 ## Returns
 
-1: The function executed successfully.
+1: This function executed successfully.
```

Comment:
"The" makes more sense

Reply:
> 
> 
> Needs a rebase, which is probably for Related Functions lists!

Blame prettifier which commited something yesterday.